src/main/java/org/example/DeliveryApp.java

```diff
@@ -0,0 +1,102 @@
+package org.example;
+import java.io.File;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+/**
+ * Small logistics application that simulates package delivery.
+ */
+public class DeliveryApp {
+    static List<Package> packages = new ArrayList<>();
+
+    public static void main(String[] args) throws InterruptedException {
+        DeliveryApp app = new DeliveryApp();
```

Comment:
file reading should be a separate class/method, also the error handling for failed reads